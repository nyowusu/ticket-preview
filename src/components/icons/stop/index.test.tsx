import { shallow } from 'enzyme';
import React from 'react';

import Component from './index';

describe('StopIcon', () => {
  it('renders', () => {
    expect(shallow(<Component type="bus" />)).toMatchSnapshot();
  });
});
