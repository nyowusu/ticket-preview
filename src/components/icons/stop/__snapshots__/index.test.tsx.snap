// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Close renders 1`] = `
<svg
  viewBox="0 0 32 32"
>
  <g
    stroke="#666666"
    strokeWidth="2"
  >
    <path
      d="M6,6 L26,26"
    />
    <path
      d="M6,6 L26,26"
      transform="translate(16.000000, 16.000000) rotate(-270.000000) translate(-16.000000, -16.000000) "
    />
  </g>
</svg>
`;
